packages/compass-aggregations/src/modules/saved-pipeline.js

```diff
@@ -73,7 +73,7 @@ export const getSavedPipelines = () => {
 export const getDirectory = () => {
   const { remote } = require('electron');
   const path = require('path');
-  const userDataDir = remote.app.getPath('userData');
+  const userDataDir = remote ? remote.app.getPath('userData') : process.env.MONGODB_COMPASS_AGGREGATE_TEST_BASE_PATH;
```

Comment:
A really silly thing, feel free to merge without changing it, but could we align the name of the variable with the plugin name?

```suggestion
  const userDataDir = remote ? remote.app.getPath('userData') : process.env.MONGODB_COMPASS_AGGREGATIONS_TEST_BASE_PATH;
```